src/lib.rs

```diff
@@ -17,14 +17,37 @@
 //!
 //! # Group Operations
 //!
-//! Groups are one of the many differentiating features of the IronCore platform. This SDK allows for easy management of your cryptographic
-//! groups. Groups can be created, updated, and deleted along with management of a group's administrators and members.
+//! Groups are one of the many differentiating features of the IronCore platform. Groups are collections of users who share access permissions.
```

Comment:
Here's some additional text that may be useful here. This comes from the DataControl Guide.

> Groups are collections of users. A user in the context of a group is called a member. Groups can have any number of members. Group membership can be modified by users designated as group administrators. Adding a user to a group is a series of cryptographic operations involving the administrating user’s keys, the group’s keys, and the new member’s public key.

Reply:
I added some of those parts to this section. I could use feedback on the wording and content of this section if you have any.

Reply:
```suggestion
//! Groups are one of the many differentiating features of the DataControl platform. Groups are collections of users who share access permissions.
```